plugins/woocommerce/src/Internal/Admin/Loader.php

```diff
@@ -222,26 +221,6 @@ public static function remove_notices() {
 		}
 	}
 
-	/**
-	 * Runs before admin notices action and hides them.
-	 */
-	public static function inject_before_notices() {
-		if ( ! PageController::is_admin_or_embed_page() ) {
-			return;
-		}
-
-		// Wrap the notices in a hidden div to prevent flickering before
-		// they are moved elsewhere in the page by WordPress Core.
-		echo '<div class="woocommerce-layout__notice-list-hide" id="wp__notice-list">';
```

Comment:
I think (requires testing) if we remove this, then we will probably completely disable all of the admin-notices-cleanup work that WC Admin does (e.g. any notices injected by any plugin will come back on all screens). JITMs do also support being inserted into a different container (`<div id="jp-admin-notices" />`) so maybe we'd be better off keeping the existing cleanup, and then carefully placing that container?

Reply:
Hi @beaulebens, thank you for your comment,

I've been working on this PR. Since the class `woocommerce-layout__notice-list-hide` essentially only adds a `display none` (so it's not being shown), I removed the class and replaced the div id with `jp-admin-notices`, otherwise, we would add the JITMs twice and hide one of them.

I couldn't find a scenario where that change breaks the experience, but if we can find one I'll leave that div as it is now and add a new one with the id `jp-admin-notices`.